### PR TITLE
Fix appimage build on Linux

### PR DESCRIPTION
**Pull request by tk-nguyen**
Need some more testing with different distros: Fedora, Debian, Ubuntu, Centos, Arch, ...

Closes: #674

### CODE REVIEW

File: appimage/AppImageBuilder.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - cp ./assets/logo.svg AppDir/usr/share/icons/hicolor/64x64/apps/twake.svg
+version: 1
+AppDir:
+  path: ./AppDir
+  app_info:
+    id: com.linagora.linux.twake
+    name: Twake
+    icon: twake
+    version: latest
+    exec: fluffychat
```
Comment (hoangdat) [inline appimage/AppImageBuilder.yml:15]: can not change it?

Comment (tk-nguyen) [reply appimage/AppImageBuilder.yml:15]: Not until you change package name in `pubspec.yaml` to `twake`: https://github.com/linagora/twake-on-matrix/blob/04967465d5ec643d58d3e46e1ef366598b525c81/pubspec.yaml#L1

Comment (hoangdat) [reply appimage/AppImageBuilder.yml:15]: okie. got it

**Comment by hoangdat** [comment]
is it ok for merge? @tk-nguyen

**Review by drminh2807 — APPROVED** [review]

**Pull request merged**